migrations/versions/aa8a94b8e115_.py

```diff
@@ -0,0 +1,64 @@
+"""empty message
+
+Revision ID: aa8a94b8e115
+Revises: 08591dac6d5c
+Create Date: 2019-10-02 16:55:20.360069
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = "aa8a94b8e115"
+down_revision = "08591dac6d5c"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.execute(
+        """
+        DELETE from 
```

Comment:
Fancy query you got here 🙂 Maybe add a comment explaining that it will delete all but the most recently-updated downloadable file for each `object_url`?